dassl/modeling/backbone/__init__.py

```diff
@@ -22,6 +22,7 @@
     shufflenet_v2_x0_5, shufflenet_v2_x1_0, shufflenet_v2_x1_5,
     shufflenet_v2_x2_0
 )
+from .resnet_dynamic import *
```

Comment:
try to avoid using `*`

Reply:
I have added `__all__` to the source file, so only the parts where `__all__` is declared will be imported.

https://github.com/KaiyangZhou/Dassl.pytorch/pull/43/files#diff-d7eabb476b680df417ee0042708ce5ec179cdb63cd37194c94481a338a35612fR17-R24

```
__all__ = [
    'resnet18_dynamic', 'resnet50_dynamic', 'resnet101_dynamic',
    'resnet18_dynamic_ms_l123', 'resnet18_dynamic_ms_l12',
    'resnet18_dynamic_ms_l1', 'resnet50_dynamic_ms_l123',
    'resnet50_dynamic_ms_l12', 'resnet50_dynamic_ms_l1',
    'resnet101_dynamic_ms_l123', 'resnet101_dynamic_ms_l12',
    'resnet101_dynamic_ms_l1'
]
```

Reply:
I think this scheme would be better? What do you think?

Reply:
yes, looks better!

could you also change single quotes to double quotes so the style is consistent throughout the code? thanks